server/container_stats.go

```diff
@@ -11,6 +11,27 @@ import (
 	pb "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
 )
 
+// ContainerStats returns stats of the container. If the container does not
```

Comment:
Same here -- I would not move the code around for the sole reason to be declared above its use.